docs/DeveloperGuide.md

```diff
@@ -317,6 +325,7 @@ Priorities:
 | `* * *`  | user           | view tasks by contact    | view tasks belonging to a contact                                          |
 | `* *`    | user           | update a task            | update the particulars of a task                                           |
 | `* * *`  | user           | delete a task            | remove tasks that I no longer need                                         |
+| `* *`    | user           | list all tasks   | to locate details of all the tasks immediately                 |
```

Comment:
Maybe you can shift the alignment to match the other lines just for readability :)
```suggestion
| `* *`    | user           | list all tasks        | to locate details of all the tasks immediately                              |
```